client/src/util/data_layer_util.ts

```diff
@@ -28,13 +33,33 @@ export const formatLegendBucket = (legend: LegendInfo | undefined): Array<Legend
   return legend?.buckets ?? new Array<LegendBucketData>();
 };
 
+/**
+ * Return the [GeographicLevel] for a given zoom value.
+ * @param legends: the map of legends available
+ * @param zoom: zoom level. Each geographic level enum is inherently
+ * assigned to an enum value.
+ */
+const getLegendForZoomLevel = (
+  legends: Map<GeographicLevel, LegendInfo>,
+  zoom: number,
+): LegendInfo | undefined => {
+  // Object.values().reverse() gives
+  // [9, 6, 5, 4, 0, "Parcel", "Zipcode", "County", "State", "Unknown"]
```

Comment:
nit: this comment is confusing - object.values gives the enum values and string values in a list?

Reply:
Removed!